index.md

```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 
 **If your company has domain experts who are writing code, [get in touch](mailto:hi@tkiley.co.uk) today to increase both the reliability of the software you produce, as well as the speed at which you can develop it.**
 
-You can read more about what the course covers and how it is structured [here](course.html). 
+I have been writing code for ten years in a range of industries including science, technology, games and healthcare. In this time I have learnt the importance of good software practises like writing automated tests to check the code is correct and using continuous integration.
 
-Get in touch today to discuss getting these seminars given to you. Email me [hi@tkiley.co.uk](mailto:hi@tkiley.co.uk) or ring me on 07948 284729.
+Writing code is now an essential part of all science and technology research and development - yet many of the scientists and domain experts have never received training on software engineering best practices, and are instead expected to figure it out. The result of this is, at best, slow software development, and at worst incorrect research results.
+
+Through the techniques covered in these seminars, not only will the code produced be more reliable, but development will actually be easier and faster for the experts.
+
+**Get a tailored seminar series that will meet the specific needs of your company.**
+
+The seminars themselves can be tailored to the specific needs of your company. I am also available following the talks to have group or one-on-one meetings to discuss the specific software engineering challenges you face, and work with you to come up with a solution that works for everyone.
```

Comment:
Intro the seminars before referencing them? "I can deliver 4 hour-long seminars..."